source/documentation/contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,100 @@
+[[contributing-user-stories]]
+= Contribute user stories to oVirt documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for oVirt GitHub repository
+
+toc::[]
+
+== Modularization backstory
+oVirt docs are modularized.
+All existing content has been replaced with content that is based on user stories and
+complies with the modularization guidelines. All future content must both
+support a user story and be modular.
+
+== How do I contribute modularized content?
+To contribute modularized content, you need to write a user story, create
+documentation modules to support the user story, and create an assembly for the
+story.
+
+== What if I don't want to write in modules?
```

Comment:
Recommendation: drop this section. It undercuts what we just said.  I suggest adding a sentence such as "To make a suggestion for improving existing content, reach out to the oVirt documentation team at <some link>." to the previosu section.

Reply:
Well, if someone wants to make a contribution and sees this section, they might just skip the whole thing. Having the heading there make it more visible, I don't want to discourage someone.